installer/conf/td-agent-bit.conf

```diff
@@ -14,6 +14,15 @@
     Path_Key filepath
     Skip_Long_Lines On
 
+[INPUT]
+    Name tail
+    Tag oms.container.log.flbplugin.*
+    Path /var/log/containers/omsagent*.log
+    DB /var/opt/microsoft/docker-cimprov/state/fblogs.db
```

Comment:
do you know if this is best practice (specifying & using and same state db  for 2 input tails?). we need to make sure this telemetry enablement does not impact log collection. i would rather use a different db .

Reply:
if you look at the db, it is basically a set of files, along with the last position that was read. So, in my opinion, should be ok. that being said, will do more testing to see if there is any impact to log collection